consensus/src/consensus_observer/network/observer_message.rs

```diff
@@ -364,13 +396,19 @@ impl PayloadWithProofAndLimit {
 /// The transaction payload of each block
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub enum BlockTransactionPayload {
-    InQuorumStore(PayloadWithProof),
-    InQuorumStoreWithLimit(PayloadWithProofAndLimit),
+    // TODO: deprecate InQuorumStore* variants
+    DeprecatedInQuorumStore(PayloadWithProof),
+    DeprecatedInQuorumStoreWithLimit(PayloadWithProofAndLimit),
     QuorumStoreInlineHybrid(PayloadWithProofAndLimit, Vec<BatchInfo>),
     OptQuorumStore(
         PayloadWithProofAndLimit,
         /* OptQS and Inline Batches */ Vec<BatchInfo>,
     ),
+    QuorumStoreInlineHybridV2(PayloadWithProofAndLimits, Vec<BatchInfo>),
+    OptQuorumStoreV2(
+        PayloadWithProofAndLimits,
+        /* OptQS and Inline Batches */ Vec<BatchInfo>,
+    ),
```

Comment:
On second thought, we don't need a V2 for opt quorum store because we haven't turned it on yet?